frontend_tests/node_tests/hotkey.js

```diff
@@ -317,6 +317,10 @@ function stubbing(func_name_to_stub, test_function) {
     assert_mapping('left_arrow', 'lightbox.prev');
     assert_mapping('right_arrow', 'lightbox.next');
 
+    hotkey.is_editing_stream_name = return_true;
+    assert_unmapped('down_arrow');
+    assert_unmapped('up_arrow');
```

Comment:
I think we should be setting `hotkey.is_editing_stream_name = return_false;` at the end of this block, right?  Seems like we have a few places where we're not doing that properly (e.g. lightbox, above)